.circleci/config.yml

```diff
@@ -669,9 +673,16 @@ jobs:
             TEST_FILES=$(echo "$TEST_FILES" | circleci tests split --split-by=timings)
             TEST_FILES=$(echo "$TEST_FILES" | sed 's|^test/||')
             MATCH_PATH="./test/{$(echo "$TEST_FILES" | paste -sd "," -)}"
-            forge <<parameters.test_command>> <<parameters.test_flags>> --match-path "$MATCH_PATH"
+            if [[ "$UPGRADE_TEST" == "true" ]]; then
+              echo "Running upgrade tests"
+              export ETH_RPC_URL="https://ci-mainnet-l1-archive.optimism.io"
```

Comment:
^ getting a [429 in CI](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/74637/workflows/b5424517-0830-41fc-aea9-65ee3f0ba7fd/jobs/3043977?invite=true#step-111-102757_18) on this URL